empathyco/x
# feat(facets-next): add all remaining components to Facets Next X-Module

Add `ClearFilters`, `ExcludeFiltersWithNoResults`, `SortedFilters`, `FiltersSearch`,
`SlicedFilters`, `SelectedFilters` and  `SelectedFiltersList` components.

EX-4615

# Pull Request Template #

Add all pending components to `NextFacets` X-Module. (Except `HierarchicalFIlter`, which has its own PR).
Also used these new components in the `LayoutNext` view to check them working.


## Motivation and Context ##

The motivation is to move all pending things from the old `Facets` X-Module to the new `FacetsNext`X-Module, before removing the old and replace it with the new.

## How to test ##
Use the `LayoutNext` view to check it working.

**REMARK:** All the facets are multiselect by default. The `MultiSelectFilters` component disappears. Now it is a modifier in the Entities which modifies the facet to be "Single Select":

```
      FilterEntityFactory.instance.registerFilterModifier('brand_facet', [SingleSelectModifier]);
```

As in Juguettos the facets are not configured as multiselected in back, and we don't do the "OR" transform in the adapter any more, some filters will return no results if you select more than one.

We will keep this way at the moment until we mock all the tests and move to the Platform Backend in the X-Components project.
